server/atelier-queries.js

```diff
@@ -71,7 +71,8 @@ const getCurrentProductBundle = (productId, headers) => {
       return {
         ...responses[0],
         styles: [ ...responses[1] ],
-        related: [ ...responses[2] ],
+        // This is to remove any duplicates from the related products array
```

Comment:
Are there duplicates because of the quality of the data received from the API, or something with the filtering within the client?

Reply:
I have found duplicates in one of the pulls, and they showed double in the carousel, so this was to mitigate that.